src/actions/PostActionCreators.js

```diff
@@ -5,7 +5,11 @@ var sectionIdsCursor = store.select('models', 'sectionIds');
 var { rawDataToPost } = require('../utils/Post');
 
 var urlBuilder = (sectionName) => {
-  return `http://www.dukechronicle.com/section/${sectionName}.json`;
+    if (sectionName !== 'frontpage') {
```

Comment:
Actually, can you please change these tabs to spaces? Use two spaces for a tab, instead of a tab character.